arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-anytrust-chain.md

```diff
@@ -0,0 +1,149 @@
+
+These parameters tailor the chain to your specific requirements and operational context.
+
+The Orbit SDK has a helper to execute this part:
+
+For an easier config preparation, the Orbit SDK provides the `prepareChainConfig` API, which takes config parameters as arguments and returns a `chainConfig` `JSON` string. Any parameters not provided will default to standard values, which are detailed in the [Orbit SDK documentation](https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/1f251f76a55bc1081f50938b0aa9f7965660ebf7/src/prepareChainConfig.ts#L3-L31).
+
+
+### 2. Deploying Your Anytrust chain
+
+After configuring your chain with the `createRollupPrepareConfig` API, the next step is to use the `createRollupPrepareTransactionRequest` API. This API is designed to take the parameters defined in the `RollupDeploymentParams`, along with the configuration generated by the `createRollupPrepareConfig` API, to prepare a transaction request. This request is then used to invoke the `createRollup` function of the `RollupCreator` contract, which effectively deploys and initializes the core contracts of your Anytrust Orbit chain.
+
+
+   For instance, to deploy using the Orbit SDK with a Config equal to `config`, a `batchPoster`, and a set of validators such as `[validator]`, the process would look like this:
+
+   ```js
+   import { createRollupPrepareTransactionRequest } from '@arbitrum/orbit-sdk';
+
+   const request = await createRollupPrepareTransactionRequest({
+       params: {
+           config,
+           batchPoster,
+           validators: [validator],
+       },
+       account: deployer_address,
+       publicClient,
+   });
+   ```
+
+After creating the raw transaction, you need to sign and broadcast it to the network.
+### 3. Setting Valid Keyset on Parent Chain:
+
+The final step in deploying your Anytrust Orbit chain is to set up the valid keyset for your Data Availability Committee (DAC) on the parent chain. This keyset is essential for ensuring that the parent chain can verify the data from your Orbit chain. The process of generating keys and the keyset for your committee is comprehensively explained in our documentation (referenced as '#'). Once you have your keyset, it needs to be established on the `SequencerInbox` contract of your Orbit chain on the parent chain.
+
+To facilitate this, we provide an API in Orbit SDK named `setValidKeysetPrepareTransactionRequest`. This API aids in setting the keyset on the parent chain. To use this API, you need specific information that you gathered in step 3. This includes the `upgradeExecutor` and `sequencerInbox` addresses of your Orbit chain, the generated keyset for your committee, and the account of the owner.
```

Comment:
This is referencing information obtained in `Step 3` (which is this step), but in any case, I think the user doesn't have this information at this point. We should point at how they can obtain the information (probably running Step 4 first, but I don't remember if that gives you the UpgradeExecutor too)

Reply:
@GreatSoshiant defined the sequence you recommended in the initial version, so it's my bad.
I thought the flow could be made more intuitive, but it wasn't a good idea.

Reply:
Merged manually.